Assignment-2/Ques1.js

```diff
@@ -0,0 +1,9 @@
+
+// Write a program to give a random output between 1 to 10 .
+
+function getRandom(){
+   let randomNumber= Math.floor(Math.random() * 10+1);
```

Comment:
Check function name. Make it getRandomNumber . Why are we using let here ? Please go through let, var, const and check this code